packages/fetchai/contracts/erc1155/contract.py

```diff
@@ -472,10 +511,9 @@ def generate_id(self, token_id, item_id):
         final_id_int = (token_id << 128) + index
         return final_id_int
 
-    #
-    # def generate_trade_nonce(self, contract, address):
-    #     """Generate a valid trade nonce."""
-    #     trade_nonce = random.randrange(0, 10000000)
-    #     while contract.instance.functions.is_nonce_used(address, trade_nonce).call():
-    #         trade_nonce = random.randrange(0, 10000000)
-    #     return trade_nonce
+    def generate_trade_nonce(self, contract, address):
```

Comment:
already defined above

Reply:
It is a copy, I am trying to remove the helpers class.